operations-playground/README.md

```diff
@@ -47,4 +47,4 @@ docker-compose down
 ## Further instructions
 
 The playground setup and more detailed instructions are presented in the
-["Getting Started" guide](https://ci.apache.org/projects/flink/flink-docs-master/getting-started/docker-playgrounds/flink-operations-playground.html) of Flink's documentation.
+["Getting Started" guide](https://ci.apache.org/projects/flink/flink-docs-release-1.10/getting-started/docker-playgrounds/flink-operations-playground.html) of Flink's documentation.
```

Comment:
I based my PR on the master branch, which is why this diff looks like this. I'm not sure this is the best way to handle this, but since the how-to-update-playgrounds doc says that master should match the latest release branch, I decided to propose we change this accordingly.

Reply:
Yes, this makes sense. Thank you!